application/common/mail/mfa-enabled.text.php

```diff
@@ -23,6 +23,17 @@
  * @var bool   $isMfaEnabled
  */
 ?>
-Dear <?= $displayName ?>,
+Dear <?= $displayName?>,
+
+Congratulations! You have enabled 2-Step Verification on your <?= $idpDisplayName ?> Identity account. This is a great
+way to protect access your account as well keep bad guys out of our corporate systems.
```

Comment:
Same typo as in HTML version: `protect access your account` --> `protect access to your account`

Reply:
Done